### DISCUSSION TITLE
MX 77 SERIES ADDED

### DISCUSSION THREAD

**Discussion by arieslaptop** [nofeletru/UsbAsp-flash#112] [Show and tell]
<MX77L25650F id="C27519" page="256" size="33554432"/>    
      <MX77L6450F id="C27517" page="256" size="8388608"/>

**Comment by richardRTF** [comment]
HOLA. yo tengo un ch341 y no reconoce la serie MX77. en tal caso como se puede programar esa serie.

**Reply by arieslaptop** [reply]
A download fork software is available for the mx77L12850F. Additionally, is that not your number? If not, then please send me the information regarding your chipset number and programmer ID, which is displayed at the bottom of the page. https://github.com/arieslaptop/ENIT-FLASH-Ver-2.2/releases/tag/v2.2
